data/themes/darktable.css

```diff
@@ -2078,9 +2035,9 @@ treeview#delete-dialog:selected
 }
 
 /* Sliders states */
-#bauhaus-popup:hover,
-#bauhaus-combobox:hover,
-#bauhaus-slider:hover
+#bauhaus-popup *:hover,
+#bauhaus-combobox *:hover,
+#bauhaus-slider *:hover
```

Comment:
@Nilvus @TurboGit this broke slider hovering prelight. Bauhaus widgets are not containers, so they don't have children that you can refer to with *

For combobox there is now a `#bauhaus-combobox:hover` above that gets picked so would be a duplicate here.

Reply:
Thanks for noticing, I'll let @Nilvus fix that.

Reply:
Oh shit, just forgotten them. This was test when I didn't have what I wanted. As you can see, I put bauhaus-combo for example some lines above. Thanks for noticing that. Will fix it. Only bauhaus-combobox have children (bauhaus-popup especially).